cfgov/unprocessed/apps/teachers-digital-platform/js/expandable-mobile.js

```diff
@@ -0,0 +1,34 @@
+import ExpandableTransition from '@cfpb/cfpb-expandables/src/ExpandableTransition';
+
+const MOBILE_COLLAPSED_CLASS = 'o-expandable__mobile-collapsed';
+const MOBILE_WIDTH = 800;
+
+let innerWidth;
+
+function getInnerWidth() {
+  return innerWidth || window.innerWidth;
+}
+
+// For unit tests
+function setInnerWidth(width) {
+  innerWidth = width;
+}
+
+/**
+ * Alter DOM as necessary before cfpb-expandables ExpandableTransition.init()
+ * so some elements won't be expanded by default.
+ */
+function beforeExpandableTransitionInit() {
+  document.querySelectorAll( '.' + MOBILE_COLLAPSED_CLASS ).forEach( el => {
```

Comment:
I think this line is breaking the functionality of the filters in IE 11. None of the faceting works as it used to, and I see this error in the console:

```
Object doesn't support property or method 'forEach'
```

Reply:
(Affecting desktop widths, just to be clear.)

Reply:
@Scotchester I'll look into this right away. Is IE11 the baseline or (gulp) earlier versions.

Reply:
Thanks, Steve! For JS-required functionality like this, we only need to support IE 11.